daft/execution/physical_plan.py

```diff
@@ -777,7 +777,7 @@ def sort(
                 partial_metadatas=None,
             )
             .add_instruction(
-                instruction=execution_step.Sample(sort_by=sort_by),
+                instruction=execution_step.Sample(fraction=0.05, sort_by=sort_by),
```

Comment:
For the sampling code for sort, it would be better to be able to set a number like 20 rather than take a fraction. For example, if our partition is only 10 elements, we rather take all the elements than just sample 1.

Reply:
Enabled sample by fraction and sample by size for table and micropartition, so sort will use sample by size.